P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium.
+Dual-licensed under CC-by and BSD2 licences
+See the file COPYING.txt for details.
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="post" mode="add" module="cmc">
+    <desc xml:lang="en" versionDate="2022-08-09"> a written (or spoken) contribution to an ongoing
+        CMC interaction which has been composed (or recorded) by its author in its entirety as part
+        of a private activity, and has been transmitted through the internet and is made available
+        on the monitor or screen of the other parties en bloc. </desc>
+    <classes>
+        <memberOf key="model.cmc"/>
+        <memberOf key="att.global"/>
+        <memberOf key="att.cmc"/>
+        <memberOf key="att.ascribed"/>
+        <memberOf key="att.canonical"/>
+        <memberOf key="att.datable"/>
+        <memberOf key="att.docStatus"/>
+        <memberOf key="att.timed"/>
+        <memberOf key="att.typed"/>
+    </classes>
+    <content>
+        <alternate minOccurs="0" maxOccurs="unbounded">
+            <textNode/>
+            <classRef key="model.headLike"/>
+            <elementRef key="opener"/>
+            <classRef key="model.pLike"/>
+            <classRef key="model.gLike"/>
+            <classRef key="model.phrase"/>
+            <classRef key="model.inter"/>
+            <classRef key="model.global"/>
+            <elementRef key="lg"/>
+            <classRef key="model.lLike"/>
+            <classRef key="model.divBottom"/>
+        </alternate>
+    </content>
+    <attList>
+        <attDef ident="modality" usage="rec" mode="add">
+            <desc xml:lang="en" versionDate="2021-09-07">written or spoken mode</desc>
+            <datatype minOccurs="1" maxOccurs="1">
+                <dataRef key="teidata.enumerated"/>
+            </datatype>
+            <valList xml:lang="en" type="semi">
+                <valItem ident="written">
+                    <desc xml:lang="en" versionDate="2021-09-07">for stretches of text and/or multimodal elements</desc>
+                </valItem>
+                <valItem ident="spoken">
+                    <gloss xml:lang="en" versionDate="2021-09-07">for audio posts</gloss>
+                </valItem>
```

Comment:
Should video posts also be consideres as a value? I don't mean posted videos from other sources but (similar to a spoken audio post) a recorded video post. There surely are platforms that enable these?!

Reply:
I would like to see “for audio (or audio-visual) posts”, I think.

Reply:
changed in 175133ff4e7708475eea2deb4264422f897ed0c8